human_aware_rl/ppo/ppo_rllib_test.py

```diff
@@ -249,35 +250,63 @@ def test_ppo_fp_sp_yes_phi(self):
     def test_ppo_bc(self):
         # Train bc model
         model_dir = self.temp_model_dir
-        params_to_override = { 
+        params_to_override = {
             "layouts" : ['inverse_marshmallow_experiment'],
             "data_path" : None,
             "epochs" : 10
         }
         bc_params = get_bc_params(**params_to_override)
         train_bc_model(model_dir, bc_params)
-    
+
         # Train rllib model
-        config_updates = { 
-            "results_dir" : self.temp_results_dir, 
-            "bc_schedule" : [(0.0, 0.0), (8e3, 1.0)], 
-            "num_training_iters" : 20, 
-            "bc_model_dir" : model_dir, 
+        config_updates = {
+            "results_dir" : self.temp_results_dir,
+            "bc_schedule" : [(0.0, 0.0), (8e3, 1.0)],
+            "num_training_iters" : 20,
+            "bc_model_dir" : model_dir,
             "evaluation_interval" : 5,
             "verbose" : False
         }
         results = ex.run(config_updates=config_updates, options={'--loglevel': 'ERROR'}).result
-    
+
         # Sanity check
         self.assertGreaterEqual(results['average_total_reward'], self.min_performance)
-    
+
         if self.compute_pickle:
             self.expected['test_ppo_bc'] = results
-    
+
         # Reproducibility test
         if self.strict:
             self.assertDictEqual(results, self.expected['test_ppo_bc'])
 
+    def test_resume_functionality(self):
+        load_path = os.path.join(os.path.abspath('.'), 'trained_example/cramped_room/checkpoint-500')
+        # Load and train an agent for another iteration
+        results = ex_fp.run(
+            config_updates={
+                "results_dir": self.temp_results_dir,
+                "num_workers": 1,
+                "num_training_iters": 1,
+                "resume_checkpoint_path": load_path,
+                "verbose": False
+            },
+            options={'--loglevel': 'ERROR'}
+        ).result
+
+        #Test that the rewards from 1 additional iteration are not too different from the original model
+        #performance
+
+        threshold = 0.1
+
+        with open('trained_example/cramped_room/result.json') as f:
+            j = json.loads(f.readlines()[-1])
```

Comment:
Should probably use a util from `utils.py` to do this (in overcooked-ai or human_aware_rl). If one doesn't exist, we should create a `read_json` util

Reply:
haven't found in documentation any built-in function, so just moved it to utils